elasticsearch/src/main/java/org/hibernate/search/backend/elasticsearch/impl/ElasticsearchIndexManager.java

```diff
@@ -499,6 +523,16 @@ public String getActualIndexName() {
 		return actualIndexName;
 	}
 
+	@Override
+	public Analyzer createAnalyzer(String name) {
+		return new ElasticsearchAnalyzer( name );
```

Comment:
I'll dig into this a bit more.
I'm wondering though what would you expect when somebody define a field with
```
@AnalyzerDef( definition = "ec-analyzer )
private String field;
```
Where ec-analyzer is a custom analyzer defined on Elasticsearch.
Would you expect this to work only on elastic search and be ignored on the other backends?